README.md

```diff
@@ -1,8 +1,8 @@
 # The REMIND R package
 
-R package **remind**, version **36.181.1**
+R package **remind**, version **36.182.1**
 
-  
+[![CRAN status](https://www.r-pkg.org/badges/version/remind)](https://cran.r-project.org/package=remind)   [![R build status](https://github.com/pik-piam/remind/workflows/check/badge.svg)](https://github.com/pik-piam/remind/actions) [![codecov](https://codecov.io/gh/pik-piam/remind/branch/master/graph/badge.svg)](https://codecov.io/gh/pik-piam/remind)
```

Comment:
Not quite sure about these lines. Is remind on CRAN? The link seems to lead nowhere. The same seems to be true for the codecov link. Maybe remove these lines from the PR.